osu.Game/Screens/Edit/Components/RadioButtons/DrawableRadioButton.cs

```diff
@@ -71,13 +77,16 @@ protected override void LoadComplete()
         {
             base.LoadComplete();
 
-            button.Selected.ValueChanged += selected =>
+            Button.Selected.ValueChanged += selected =>
             {
                 updateSelectionState();
                 if (selected.NewValue)
-                    Selected?.Invoke(button);
+                    Selected?.Invoke(Button);
             };
 
+            editorBeatmap?.HasTiming.BindValueChanged(hasTiming => Button.Selected.Disabled = !hasTiming.NewValue, true);
```

Comment:
A little weirded out by this sort of placement-tool-specific logic living in something as generically-named as a `DrawableRadioButton`. I know it's not used for anything else, but I dunno.

Also, the selection button is getting disabled by this, which makes it extra weird in edge-cases like:

1. Add timing point
2. Select hitcircle tool
3. Remove timing point
4. Now you can't select any more, and you can still place

Reply:
Yeah, I think we can rename this.

Will add an extra check to set the tool back to select in said case, nice catch.